mailipy/gen.py

```diff
@@ -38,6 +38,8 @@
     </body>
 </html>"""
 
+csv.field_size_limit(sys.maxsize)
```

Comment:
Would it be better to use some high value defined by us?

If I understand correctly, by using `sys.maxsize` we are just letting the csv module parse forever (and crash). It's probably better to get an exception.

(Not blocking: if you decide to keep it as it is maybe add a comment/todo above it)

Reply:
Not sure which is the better option, either of the two are fine for me.

Reply:
Then I would say let's go with a reasonable limit (10MB?) that we define ourselves